config/webpack/loaders/devSiteCodeblock.js

```diff
@@ -0,0 +1,23 @@
+// const { getOptions } = require('loader-utils');
+const path = require('path');
+
+/**
+ * Load the contents of the file into the codeblock
+ * This loader expects to be chained with the mdx loader.
+ * @param {*} content
+ */
+const loader = async function loader(content) {
+  const callback = this.async();
+  // remove . from extension;
+  const extension = path.extname(this.resourcePath).slice(1);
+
+  const md = [
+    `\`\`\`\`${extension}`,
```

Comment:
I think there are one too many `s here, which could be causing the weird closing tag being seen

Reply:
good catch, this was puzzling me.